fjord/feedback/templates/feedback/generic_feedback.html

```diff
@@ -193,7 +180,14 @@
               </div>
             </div>
             <div class="input-box">
-              <button id="form-submit-btn" class="complete btn submit" disabled>{{ _('Submit feedback') }}</button>
+              <button id="form-submit-btn" class="complete btn submit" disabled>
+                {% if LANG == 'en-US' %}
+                  {# NOTE: Changing this to Submit, but have to wait for translations to be done. #}
+                  {{ _('Submit') }}
+                {% else %}
+                  {{ _('Submit feedback') }}
+                {% endif %}
+              </button>
```

Comment:
Instead of showing "Submit feedback", it was agreed upon to switch it to just "Submit". However, that's not a fully translated string, so we're limiting the new string to en-US only. I can remove this silliness when it gets translated.